app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -64,5 +65,17 @@ def stop_event_monitor_queue_on_credential_change
     def supports_cloud_tenants?
       false
     end
+
+    def sync_cloud_tenants_with_tenants
+      return unless supports_cloud_tenants?
+      sync_root_tenant
+    end
+
+    def sync_root_tenant
+      ems_tenant = source_tenant || Tenant.new(:parent => Tenant.root_tenant, :source => self)
+
+      ems_tenant_name = "#{self.class.description} Cloud Provider #{name}"
+      ems_tenant.update_attributes(:name => ems_tenant_name, :description => ems_tenant_name)
```

Comment:
what about error handling here? How about using `update_attributes!`, this would at least throw an exception